src/types/index.ts

```diff
@@ -47,7 +47,7 @@ export type ConnectUsbReaderParams = {
   autoReconnectOnUnexpectedDisconnect?: boolean;
 };
 
-export type ConnectLocalMobileParams = {
+export type ConnectTapToPayParams = {
   reader: Reader.Type;
   locationId?: string;
   onBehalfOf?: string;
```

Comment:
We should update `ConnectTapToPayParams` so that `autoReconnectOnUnexpectedDisconnect` defaults to true

Reply:
in ts layer it's optional but in native it will default to true for tap to pay now.